server/sonar-server/src/main/java/org/sonar/server/authentication/UserSessionInitializer.java

```diff
@@ -104,6 +117,14 @@ public boolean initUserSession(HttpServletRequest request, HttpServletResponse r
     }
   }
 
+  private static boolean shouldContinueFilterOnError(String path) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused private "shouldContinueFilterOnError" method. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AUnusedPrivateMethod)